fs/fs.go

```diff
@@ -75,9 +75,8 @@ import (
 )
 
 const (
-	defaultFuseTimeout    = time.Second
-	defaultMaxConcurrency = 2
```

Comment:
This change is because ``defaultMaxConcurrency`` is unused so it fails the lint style check.